AppointEase/AppointEase.Application.Contracts/Common/Co.cs

```diff
@@ -0,0 +1,34 @@
+﻿using AppointEase.Application.Contracts.Interfaces;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AppointEase.Application.Contracts.Co
+{
+    public class Common : ICommon
+    {
+        public static string Role { get; set; }
+
+        private readonly ILogger _logger;
+
+        public Common(ILogger<Common> logger)
+        {
+            _logger = logger;
+        }
+
+        public void LogInformation(string message)
```

Comment:
This shouldn't be a method, you are encapsulating and existing method and adding nothing to it, `LogInformation `is already a method from the `ILogger`. If this class is used for methods that we need throughout the application layer than  the naming should be `ApplicationExtensions`. Also why its the namespace `Co`?